ixbrlparse/core.py

```diff
@@ -248,6 +258,11 @@ def _get_numeric(self) -> None:
                         text=s.text,
                         context=self.contexts.get(context_ref, context_ref),
                         unit=self.units.get(unit_ref, unit_ref),
+                        ixt=self.namespaces.get(
+                            f'xmlns:{s.attrs["format"].split(":")[0]}', []
+                        )[0]
+                        if s.get("format") is not None
+                        else "",
```

Comment:
This needs to be a bit more robust for the case where it's not found in `self.namespaces` otherwise we'll get lots of keyerrors.